packages/@lwc/babel-plugin-component/src/post-process/dedupe-imports.js

```diff
@@ -17,6 +17,12 @@ module.exports = function dedupeImports({ types: t }, path) {
 
     importStatements.forEach((importPath) => {
         const sourceLiteral = importPath.node.source;
+
+        // If the import is of the type import * as X, just ignore it since we can't dedupe
+        if (t.isImportNamespaceSpecifier(importPath.node.specifiers[0])) {
```

Comment:
I'm not sure about this... because you have multiple other syntax that can be combined, e.g.:

```
import foo, { x } from 'foo';
import foo1, * as ns from 'foo';
```

Reply:
Added a test case for this